prog/weave-npc/main.go

```diff
@@ -32,6 +31,7 @@ var (
 	nodeName       string
 	maxList        int
 	bridgePortName string
+	stopChan       chan struct{}
```

Comment:
I think this could be a local variable in `root()`

Reply:
I didn't initially notice this.  thank you.  change made.